api/src/services/email.service.ts

```diff
@@ -326,7 +326,11 @@ export class EmailService {
     );
   }
 
-  public async sendSingleUseCode(user: User, singleUseCode: string) {
+  public async sendSingleUseCode(
+    user: User,
+    singleUseCode: string,
+    template?: string,
```

Comment:
wondering if using an enum here would make template easier to work with (less searching for what the options are if we ever use this function for something else)